### PR TITLE
added httpcheck dashboard

### PR DESCRIPTION
**Pull request by hecomp**
## Description
What does this PR do?
added httpcheck dashboard

## PR checklist

Please confirm the following items:
- [] At least one screenshot of the proposed dashboard is included in this PR. If you're proposing substantive changes to queries or new queries then please ensure your screenshot shows relevant data.
- [] All chart names are in [Title case](https://en.wikipedia.org/wiki/Title_case).
- [] All query names are lower case letters, beginning the first query of each chart as "a" and proceeding alphabetically ... "b", "c", etcetera.

Screenshot 2023-06-07 at 9 17 32 AM

### CODE REVIEW

File: collector-dashboards/otel-collector-httpcheck-dashboard/main.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+
+resource "lightstep_dashboard" "otel_collector_httpcheck_dashboard" {
+  project_name          = var.lightstep_project
+  dashboard_name        = "OpenTelemetry HttpCheck Dashboard"
+  dashboard_description = "Monitor the HTTPCheck of your OpenTelemetry Collector instances"
+
+  chart {
+    name = "HttpCheck Duration"
```
Comment (nslaughter) [inline collector-dashboards/otel-collector-httpcheck-dashboard/main.tf:18]:
```suggestion
    name = "HTTPCheck Duration"
```

File: collector-dashboards/otel-collector-httpcheck-dashboard/main.tf [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = "metric httpcheck.duration | rate | group_by [], sum"
+    }
+
+  }
+
+  chart {
+    name = "Httpcheck Status Code"
```
Comment (nslaughter) [inline collector-dashboards/otel-collector-httpcheck-dashboard/main.tf:32]:
```suggestion
    name = "HTTPcheck Status Code"
```

File: collector-dashboards/otel-collector-httpcheck-dashboard/variables.tf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+variable "lightstep_project" {
+  description = "Name of Lightstep project"
```
Comment (nslaughter) [inline collector-dashboards/otel-collector-httpcheck-dashboard/variables.tf:2]:
```suggestion
  description = "Name of Cloud Observability project"
```

**Review by nslaughter — CHANGES_REQUESTED** [review]
Requested changes as suggestions

**Comment by hecomp** [comment]
> Requested changes as suggestions

Addressed.

**Pull request merged**